specification/applicationinsights/resource-manager/readme.cli.md

```diff
@@ -15,17 +15,17 @@ cli:
       - select: 'operation'
         where:
             operationGroup: 'components'
-            operation: '^(delete|purge|get_purge_status)$'
+            operation: '^(delete|purge|getPurgeStatus)$'
```

Comment:
Should operation start with Capital Letter?

Reply:
it's case insensitive now